src/main/routes/port-forward-route.ts

```diff
@@ -58,8 +68,12 @@ class PortForward {
       }
     });
 
+    this.internalPort = await getPortFrom(this.process.stdout, {
```

Comment:
Should this be in the try/catch?

Reply:
I sort of want the error to be thrown. That way it gets passed to the renderer.